File: 2022 czerwiec/python/4.py
with open("liczby.txt", "r") as f:
    file = list(map(int, f.read().strip().split('\n')))

cnt = dict()
for i in file:
    if i not in cnt.keys():
        cnt[i] = 0
    cnt[i] += 1

print(len(cnt.keys()), sum(1 for i in cnt.values() if i == 2), sum(1 for i in cnt.values() if i == 3))

File: 2022 czerwiec/c++/4.cpp
#include <iostream>
#include <fstream>
using namespace std;

int cnt[10000];

int main()
{
    ifstream in("liczby.txt");
    
    int n;
    while (in >> n)
    {
        cnt[n]++;
    }

    int ans1 = 0, ans2 = 0, ans3 = 0;
    for (int i = 11; i < 10000; i += 2)
    {
        if (cnt[i] > 0)
            ans1++;
        
        if (cnt[i] == 2)
            ans2++;
        else if (cnt[i] == 3)
            ans3++;
    }
    cout << ans1 << ' ' << ans2 << ' ' << ans3 << '\n';
}